src/rulesets/ruleset.json

```diff
@@ -31,12 +31,12 @@
   "rules": [
     {
       "id": "AS0059",
```

Comment:
AFAIK, the way we enable errors is by deleting the entries from this file (as Error is the default)